context-free.md

```diff
@@ -2,21 +2,32 @@
 
 ##  Who is this programming language for?
 
+Context Free Art is a programing language built for people that want to design compture generated art peices. This can be anything from graphic art to desgins. I would say that the primary focus for this language is for those who want a simple way to design art online. This being simpler then Python or Java.
 
 ## What is easy to do in this language? Why is it easy?
 
+It was easy to make shapes in this language becuase all you need is the name of the shape you are trying to draw. It was also fairly easy to understand what the program was doing. For instance you start your file with startshape and you give it a variable name. After this all you have to do is define a rule for that varible such as draw or TRIANGLE. 
 
 ## What is hard to do in this language? Why is it hard?
 
+In general it was tough to decide what type of picture to make and how I was going to accoplish it. It was also difficult understanding how certian parts of the code would affect the drawing this is mostly becuase of the unfimiliarity of the program. As I got more comfortable with the laguage, I was able to accomplish more complext things like adding a backround and changing the color. 
 
 ## How did you learn how to program in this language?
 
+I learned how to program in this laguage by looking at the documentation and referncing other simple art peices that I found on the web. After refrencing these sources I found the laguange to be roughly stright forward. The peices of art that I refrenced gave me the idea to draw the Sierpinski triangle. 
```

Comment:
Were a lot of the other art pieces that you looked at very mathematical like this?